app/assets/javascripts/extractors.js

```diff
@@ -105,6 +114,34 @@ $(document).ready(function () {
         });
     });
 
+    // Try regular expression against example.
+    $(".xtrc-try-regex-replace").on("click", function () {
+        var button = $(this);
+
+        button.html("<i class='fa fa-spinner fa-spin'></i> Trying...");
+        var promise = testRegexReplace($("#regex").val(), $("#replacement").val(), $("#replace_all").is(":checked"));
+
+        promise.done(function (matchResult) {
+            if (matchResult.finds) {
+                if (matchResult.match != null) {
+                    highlightMatchResult(matchResult);
```

Comment:
I think in this case it would be more helpful to show the replacement text somewhere instead of highlighting the matching part in the original message (`matchResult.match.match` contains the text once the replacement is applied), something like we do on the grok patterns.

Let me know if you need some help with it :)